src/components/Network/NetworkGraphSearchBar/NetworkGraphSearchBar.tsx

```diff
@@ -48,21 +49,24 @@ const NetworkGraphSearchBar = ({
     const [links, setLinks] = useState<any>(null)
     const [searchResultsOpen, setSearchResultsOpen] = useState(true)
     const innerContainer = useRef(null)
+    const { width } = useWindowSize()
+    const isMobile = width <= 768
```

Comment:
Might be good if we create some kind of "useMobile" hook to check if it's mobile. I noticed we recreate this kind of function in several places